soc/sparc/gr716a/Kconfig.defconfig

```diff
@@ -13,4 +13,13 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
 	default 50000000
 
+
+if GPIO
+
+config GPIO_GRGPIO2
+	default y
+
+endif
```

Comment:
We generally prefer `depends on` rather than `if` #22666. Let's write it as
```
config GPIO_GRGPIO2
	default y
	depends on GPIO
```

Reply:
Done. Thanks!